src/shared/type-coercions.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Type conversion methods that gently coerce incoming configuration values to
```

Comment:
```suggestion
 * Type conversion methods that gently coerce incoming configuration values to
```

By "gently" are you referring to the fact that these functions don't throw if the values are primitives or object/array literals?

Reply:
ya, might be better to change that word